src/repos/NotificationRepo.ts

```diff
@@ -132,6 +136,36 @@ const notifyNewMagazines = async (magazineIDs: string[]): Promise<void> => {
   });
 };
 
+const sendNewFlyerNotification = async (
+  user: User,
+  flyer: Flyer,
+  organization: Organization,
+): Promise<void> => {
+  const notifTitle = organization.name;
+  const notifBody = flyer.title;
+
+  const uniqueData = {
+    flyerID: flyer.id,
+    flyerURL: flyer.flyerURL,
+  };
+
+  sendNotif(user, notifTitle, notifBody, uniqueData, 'new_flyer');
+};
+
+/**
+ * Send notifications for new flyers posted by organizations.
+ */
+const notifyNewFlyers = async (flyerIDs: string[]): Promise<void> => {
+  flyerIDs.forEach(async (f) => {
+    const flyer = await FlyerRepo.getFlyerByID(f); // eslint-disable-line
+    const organization = await OrganizationRepo.getOrganizationBySlug(flyer.organizationSlug);
+    const followers = await UserRepo.getUsersFollowingOrganization(flyer.organizationSlug);
+    followers.forEach(async (follower) => {
+      sendNewFlyerNotification(follower, flyer, organization);
```

Comment:
I don't see a good reason to factor out this logic into a function unless you plan to expose `sendNewFlyerNotification` like the other functions `sendNewMagazineNotification`, etc. are. I know that we also don't use those anywhere else in the codebase, but it seems like we might have been planning to at some point or something.

For now I'd say you can just put the logic of `sendNewFlyerNotification` inside the ForEach and delete the additional function, since it makes the logic harder to follow

Reply:
Looking at the code, it seems like `sendNewMagazineNotification` and `sendNewArticleNotification` are exported but not actually used anywhere. Should we put this logic in their respective `forEach` or do we just keep it as is? Whatever approach we use is fine as long as we keep it consistent.

Reply:
Yeah let's just inline all of those functions then